pkg/NAMESPACE

```diff
@@ -97,3 +103,5 @@ export("jsonFile",
        "uncacheTable")
 
 export("sparkRSQL.init")
+export("print.structType",
```

Comment:
Is it possible to override the behavior of `str(structType)` without export these two methods?

For exported API, less is better,